### PR TITLE
Add Valor

### PR DESCRIPTION
**Pull request by Xelus22**
## Description

Add Valor Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* Add Valor Keyboard

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xelus/valor/readme.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+An Alice replacement keyboard PCB.
+
+* Keyboard Maintainer: [Xelus22](https://github.com/Xelus22)
+* Hardware Supported: CONSTELLATION
+* Hardware Availability: Custom keyboard group buys
+
+Make example for this keyboard (after setting up your build environment):
+
+    make aeboards/constellation:default
```
Comment (zvecr) [inline keyboards/xelus/valor/readme.md:11]: `aeboards/constellation`?

**Review by zvecr — CHANGES_REQUESTED** [review]
Few other things
1. Copyright wilba 2 years ago?
2. Not a massive fan of the board name being in the layout macro.

File: keyboards/xelus/valor/rules.mk [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+RGBLIGHT_ENABLE = yes
```
Comment (fauxpark) [inline keyboards/xelus/valor/rules.mk:22]:
```suggestion
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by noroadsleft** [comment]
> 2\. Not a massive fan of the board name being in the layout macro.

100% agree.

File: keyboards/xelus/valor/readme.md (hunk 1 above)
Comment (Xelus22) [reply keyboards/xelus/valor/readme.md:11]: Sorry, I just copy pasted this.  Forgot to change it over.

File: keyboards/xelus/valor/config.h [hunk 3]
```diff
@@ -0,0 +1,53 @@
+/* Copyright 2018 Jason Williams (Wilba)
```
Comment (noroadsleft) [inline keyboards/xelus/valor/config.h:1]: Blink twice if Wilba is holding you hostage.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**